.github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: 🚨 Please make sure to fill out the template carefully. Otherwise, we may not be able to assist you. 🚨
+title: "[Feature]: "
+labels: ["✨ feature", "📋 needs review"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this feature request. 🚀
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: ✏️ Problem Description
+      description: Describe the problem or lack of functionality that you want to address. ✨
+      placeholder: What is not working as desired? Which feature is missing or could be improved?
+    validations:
+      required: true
+  - type: textarea
+    id: feature-request
+    attributes:
+      label: 💡 Feature Request
+      description: What feature do you want to request or improve? 🔍
+      placeholder: Please describe the desired feature in detail.
+    validations:
+      required: true
+  - type: textarea
+    id: context
+    attributes:
+      label: 🌍 Context
+      description: Describe the context or background in which the new feature or improvement would be useful. 📝
+      placeholder: Which user group or target audience would it affect? What benefits would they gain from it?
+  - type: textarea
+    id: additional-info
+    attributes:
+      label: 🔍 Additional Information
+      description: Include any relevant information that could help the developers better understand the problem and find a solution.
```

Comment:
Suggestion: 'your request'

Reply:
What part should be replaced by "your request"?

Reply:
The suggestion is to replace 'the problem' by 'your request'

Reply:
Yeah, that is much better. I will change that.